pxr/usdImaging/usdImagingGL/CMakeLists.txt

```diff
@@ -142,3 +146,4432 @@ pxr_build_test(testUsdImagingGLSurfaceShader
+        testUsdImagingGLFaceVaryingLinearInterp_Adaptive_Medium.png
+    FAIL 1
+    FAIL_PERCENT 1
+    PERCEPTUAL
+    EXPECTED_RETURN_CODE 0
+    TESTENV testUsdImagingGLFaceVarying
+    ENV
+        HD_ENABLE_OPENSUBDIV3_ADAPTIVE=1
+)
+
+
+pxr_register_test(testUsdImagingGLFaceVaryingLeftHanded_Adaptive_Low
+    COMMAND "${CMAKE_INSTALL_PREFIX}/tests/testUsdImagingGLBasicDrawing -complexity 1.1 -offscreen -lighting -camlight -shading wireOnSurface -frameAll -stage testUsdImagingGLFaceVaryingLeftHanded.usda -write testUsdImagingGLFaceVaryingLeftHanded_Adaptive_Low.png"
+    IMAGE_DIFF_COMPARE
+        testUsdImagingGLFaceVaryingLeftHanded_Adaptive_Low.png
+    FAIL 1
+    FAIL_PERCENT 1
+    PERCEPTUAL
+    EXPECTED_RETURN_CODE 0
+    TESTENV testUsdImagingGLFaceVarying
+    ENV
+        HD_ENABLE_OPENSUBDIV3_ADAPTIVE=1
+)
+
+pxr_register_test(testUsdImagingGLShadingPrimvars
+    COMMAND "${CMAKE_INSTALL_PREFIX}/tests/testUsdImagingGLBasicDrawing -offscreen -lighting -shading smooth -frameAll -stage testPrimvarNameHandlingInGL.usda -write testUsdImagingGLShadingPrimvars.png"
+    IMAGE_DIFF_COMPARE
+        testUsdImagingGLShadingPrimvars.png
+    FAIL 1
+    FAIL_PERCENT 1
+    PERCEPTUAL
+    EXPECTED_RETURN_CODE 0
+    TESTENV testUsdImagingGLShadingPrimvars
+    ENV
+        HD_ENABLE_OPENSUBDIV3_ADAPTIVE=1
+)
+
+# The following tests are disabled since the stages are unable to load any .tx files
+# In theory these image files should be able to load through the hioOiio plugin but I have not
+# been able to get that too work. I did try to convert these images through OIIO's iconvert tool
+# but it also fails trying to open the .tx files
+#pxr_register_test(testUsdImagingGLTextureWrap
+#    COMMAND "${CMAKE_INSTALL_PREFIX}/tests/testUsdImagingGLBasicDrawing -offscreen -lighting -shading smooth -frameAll -stage textureWrap.usda -write textureWrap.png"
+#    IMAGE_DIFF_COMPARE
+#        textureWrap.png
+#    FAIL 1
+#    FAIL_PERCENT 1
+#    PERCEPTUAL
+#    EXPECTED_RETURN_CODE 0
+#    TESTENV testUsdImagingGLTextureWrap
+#    ENV
+#        HGIGL_ENABLE_BINDLESS_TEXTURE=1
+#)
+#
+#pxr_register_test(testUsdImagingGLTextureWrapNonBindless
+#    COMMAND "${CMAKE_INSTALL_PREFIX}/tests/testUsdImagingGLBasicDrawing -offscreen -lighting -shading smooth -frameAll -stage textureWrap.usda -write textureWrap.png"
+#    IMAGE_DIFF_COMPARE
+#        textureWrap.png
+#    FAIL 1
+#    FAIL_PERCENT 1
+#    PERCEPTUAL
+#    EXPECTED_RETURN_CODE 0
+#    TESTENV testUsdImagingGLTextureWrap
+#    ENV
+#        HGIGL_ENABLE_BINDLESS_TEXTURE=0
+#)
+
+pxr_register_test(testUsdImagingGLUsdLux
+    COMMAND "${CMAKE_INSTALL_PREFIX}/tests/testUsdImagingGLBasicDrawing -offscreen -lighting -shading smooth -frameAll -stage usdLux.usda -write usdLux.png"
+    IMAGE_DIFF_COMPARE
+        usdLux.png
+    FAIL 1
+    FAIL_PERCENT 1
+    PERCEPTUAL
+    EXPECTED_RETURN_CODE 0
+    TESTENV testUsdImagingGLUsdLux
+)
+
+# TODO: Requires create_symlinks.py to run properly
```

Comment:
The testSpecs.xml reference this `create_symlinks.py` script. I'm not sure how this exactly sets up the symlinks so the following test is disabled for now. It might be worth asking about the script so we can enable this test